### ISSUE TITLE
Pose Transfer seems incomplete

### ISSUE THREAD

**Issue by DrCyanide** [cgtinker/BlendArMocap#157]
I have the same issue with videos, but for the sake of simple reproducibility here's a photo I'm using as the reference pose. (OpenCV supports image sequences, and this is just an image sequence with length of one!)
![young-woman-sitting-floor-looking-up-smiling-side-view-full-length-studio-shot-isolated-white-smiling-young-woman-99713426](https://user-images.githubusercontent.com/13542136/237004476-86dbaaeb-82c8-4616-beb0-558db06e1e83.jpg)

The pose detection seems good
![image](https://user-images.githubusercontent.com/13542136/237005006-c491c6d3-f597-41ba-8b26-5c7509fb6129.png)

And the points representing the pose seems to be lined up in Blender. At this point, it _feels_ like it should be smooth sailing.
![image](https://user-images.githubusercontent.com/13542136/237005199-5f81658f-70d7-4624-8ffe-85f7ffa8dad4.png)

However, when I click Transfer Animation, the results are pretty bad.
![image](https://user-images.githubusercontent.com/13542136/237005484-ff90487d-5bc9-4cfc-ab2f-785bdd1934ad.png)

It's more obvious in video tests, but the wrists and ankles feel "pinned" in place - they'll drift a bit with the overall model but never really move up, down, in, or out from the body.
Even when I lift my hands over top my head in webcam mode the transferred wrists stay at about that mid-chest level that they are in this screenshot.

The model I'm using here to demonstrate _should_ be using the Rigify rig, although I admit to not knowing what version of the rig. I've tried it with the un-modeled Human Meta rig, generating the rig from that, and get similar results. With the unmodeled rig it's harder to tell what's going on (I don't know what settings you clicked in your video to show the rig as anything other than outlines). I'm fine re-testing with the empty rig if someone can tell me how to make it show up.

Using BlendARMocap 1.6.0 and Blender 3.4.1

**Comment by cgtinker** [comment]
There are many tools out there which use rigify, but the rigs they use NOT based on the metarig... which leads to confusion. 

Most likely a new transfer setup can be generated for them. I described the process [here](https://cgtinker.github.io/BlendArMocap/transfer/create_config.html). Also planning to make a video soon (simple workflow example on how to transfer to other rigs).

Can you give the steps above a try and let me know?

**Comment by DrCyanide** [comment]
I set `Display as: Solid` in the Object Properties just like you have in your screenshot, but it still shows as just the wireframe. I tried changing the Viewport Display settings under Object Data Properties, but none of them made it show either. There's probably some other setting that I'm missing.

![image](https://github.com/cgtinker/BlendArMocap/assets/13542136/9e1fa278-9166-4b37-8f54-3509092e8b42)

This crumpled mess is using the generated, non-upgraded rig. 

I tested it again on another computer I have which is running Blender 3.3.1 and BlendARMocap 1.6.0 and got the same results (both with the display solid and the crumpled mess)

**Comment by DrCyanide** [comment]
[Deformed Sitting.zip](https://github.com/cgtinker/BlendArMocap/files/11431888/Deformed.Sitting.zip)
Here's both the blend file and the saved config for that pose. 

I'm guessing there's something different in a version of the plugin I'm using. Here's screenshots of both my Rigify and BlendARMocap settings.
![image](https://github.com/cgtinker/BlendArMocap/assets/13542136/be45b39d-0855-4929-b0b9-bbbd6e73a728)

![image](https://github.com/cgtinker/BlendArMocap/assets/13542136/a6303743-9c2f-4faf-a35f-bdd6210b3299)

**Comment by cgtinker** [comment]
It seems you tried to do the detection on a jpg file - I think that's the root of the issue. Sometimes mediapipes detection has to pick up on some motion to produce accurate results, showing an image for a long time is not the same as motion. However, it may produces better results than just an image.

Please use as sample this .mp4 for reference if that fixes the issue.
Note: Check the first and something like frame 10, it's often the case that the first x-frames deliver garbage.

https://github.com/cgtinker/BlendArMocap/assets/55130054/f0d7e1a4-b7af-4ea6-84b6-7d0d2dc4e563

**Comment by DrCyanide** [comment]
OK, that's working now! After a few frames, the character is in the siting position. I must have had really bad luck between my webcam tests and sample videos. I was trying to do holistic on the webcam, but could only ever get my upper body in the shot.

Sorry if testing with still image ended up being a bit of a detour. I'm not looking to make an animation, but to pose several characters quickly for a comic. I thought having a handful of pictures to chose from would be faster and less intensive than a video trying out a bunch of poses, but I guess not.

**Comment by inegvin** [comment]
I've got the same issue as DrCyanide mentions in his first post. It happens in the scene where BlendArMocap has already been running. So apparently the issue appears due to incorrect cleaning up of the previous animation tests. I used a new file with the same pose reference, and it works perfectly fine. But it would be great to know how to clean up the scene from all possible changes that plugin has brought, to be able to start the animation process over.
